bestofjs/bestofjs
# Client bundle cleanup + Next.js 14.0.3 upgrade

## Goal

Following the introduction of Webpack bundle analyzer (see #245 ) I noticed that the library `slugify`, used to generate projects slugs included in the URLs, was included in the bundle. It should be used only in the server-side.

This PR remove an unused function, that was causing `slugify` package to be included in the client bundle.

Impact on the bundles (`First Load JS`)

- before: `120 kB`
- after: `117 kB`

Also, upgrading to Next.js 14.0.3 seems to have a positive effect. After the upgrade: `114 kB`

## How to test

- Run the analyze command
- Check the total amount of JS: 114 kB
- check the client bundle (`client.html`)
- search for `slugify` in the "search modules" field: it should say "nothing found"

## Screenshots

![image](https://github.com/bestofjs/bestofjs/assets/5546996/fe40c7b0-51e4-4193-a6da-5d533ad2bcf2)

![image](https://github.com/bestofjs/bestofjs/assets/5546996/3f3bd848-a89f-4431-a543-95f2fd849d4b)
